pages/infoPage/index.js

```diff
@@ -7,7 +7,24 @@ function InfoPage() {
 
   return (
     <InfoContainer>
-      <h3>Coming soon...</h3>
+      <h3>82nd ANNUAL MEETING</h3>
+      <h4>The Westin Harbour Castle - Toronto, Ontario, Canada</h4>
+      <h4>November 2-5, 2022</h4>
+      <h4>HOST COMMITTEE</h4>
+      <p>David C. Evans, Co-Chair Mary T. Silcox, Co-Chair</p>
+      <h4>SYMPOSIUM CONVENORS</h4>
+      <p>
+        Nina G. Jablonski, Xueping Ji, Taormina Lepore, David A. Levering,
+        Denise F. Su
+      </p>
+      <h4>PROGRAMM COMMITTEE</h4>
+      <p>
+        Ken D. Angielczyk, Victoria Arbour, Arnau Bolet, Matthew R. Borths,
+        Jennifer Botha, Judd A. Case, Kimberley Chapelle, Karen Chin, Brian
+        Choo, Thomas M. Cullen, Liping Dong, Alton C. Dooley, Jaelyn J. Eberle
+      </p>
+      <h4>EDITORS</h4>
+      <p>Dana J. Ehret, Andrew A. Farke, Ethan Fulwood</p>
       <NavButton type="button" onClick={() => router.push("/landingPage")}>
         Home
       </NavButton>
```

Comment:
As you are redirecting the user to another page here, this should be a `Link` instead of a `button`.

Reply:
What exactly do you mean by Link? I had `Link` before but switched to `useRouter` for page navigation. Should I reverse this?

Reply:
Yes, correct. You can use the `Link` component to simply link to the other page. You can style the `Link` as a button if you still want it to look like one. 
You can have a look at the FishShop from the **Next.js Routing** sessions :)